dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -29,21 +30,24 @@ def funometer_by_script
       ratings = PuzzleRating.where('puzzle_ratings.script_id = ?', @script_id)
       @overall_percentage = get_percentage_positive(ratings)
 
-      # Generate the funometer percentages for the script, by day, for the last month.
-      @ratings_by_day_headers = ['Date', 'Percentage', 'Count']
-      @ratings_by_day, percentages_by_day = get_ratings_by_day(ratings)
+      # Generate the funometer percentages for the script, by day.
+      @ratings_by_day = get_ratings_by_day(ratings)
```

Comment:
It would be great if the dual sources for this data are at the very least explicitly documented somewhere, or even more ideally if the cronjob was able to make use of `get_ratings_by_day`.